src/slashers/VetoableSlasher.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {IStrategy} from "eigenlayer-contracts/src/contracts/interfaces/IStrategy.sol";
+import {SlasherBase} from "./SlasherBase.sol";
+
+contract VetoableSlashing is SlasherBase {
+    struct SlashingRequest {
+        address operator;
+        uint32 operatorSetId;
+        IStrategy[] strategies;
+        uint256 wadToSlash;
+        string description;
+        uint256 requestTimestamp;
+        SlashingStatus status;
+    }
+
+    uint256 public constant VETO_PERIOD = 3 days;
```

Comment:
immutable

Reply:
constant feels fine imho. what do you view the benefit of the immutable vs constant?